pkg/schema/metric_names.go

```diff
@@ -16,9 +16,8 @@
  *  limitations under the License.
  */
 
-package events
+package schema
 
-const (
-	EventReasonNodeIpUpdate   = "NodeIpUpdate"
-	EventReasonNodePortUpdate = "NodePortUpdate"
+var (
+	MetricNetPolViolation = "net_pol_violation"
```

Comment:
We need to prefix all metric names with `kubeslice_` to make it easily distinguishable in prometheus

Reply:
is `events_enum.go` renamed to this file? maybe github diff is wrong

Reply:
It will by default prefixed by the namespace. Shall we explicitly prefix again? For `kubeslice_*` kind of namespaces, it might be duplicate then.

Reply:
The github diff is wrong. I had removed `events_enum.go` file entirely since we have `schema/event_names.go` now.